src/openrct2/network/NetworkGroup.h

```diff
@@ -32,15 +32,19 @@ class NetworkGroup final
      * @return A NetworkGroup object
      * @note json is deliberately left non-const: json_t behaviour changes when const
      */
-    static NetworkGroup FromJson(json_t& json);
+    static NetworkGroup FromJson(const json_t& json);
+
+    constexpr const std::string& GetName() const noexcept
+    {
+        return _name;
+    }
```

Comment:
Move this back to cpp, also makes little sense to have constexpr as this all happens at runtime.

Reply:
Reverted constexpr